nservicebus/scaling.md

```diff
@@ -4,38 +4,36 @@ summary: NServiceBus provides several options to scale out a system
 redirects:
  - transports/scale-out
  - nservicebus/architecture/scaling
-reviewed: 2021-07-23
+reviewed: 2024-06-17
 ---
 
-This page describes how to scale out endpoints using NServiceBus. There are several reasons to scale out, such as to achieve higher message throughput, or to provide high availability.
-
-It is important to distinguish between a logical endpoint and an endpoint instance. Review the documentation on [logical endpoints](/nservicebus/endpoints/) for more details.
```

Comment:
reason for removing?

Reply:
I was going back and forth on this one but ultimately felt that this is schooling people and I don't like to be schooled. So I figured by linking to the appropriate definition, it should be good enough for people to follow through. I'm also not opposed to bringing back the wording if we prefer that.